app/test/house-component-test.js

```diff
@@ -0,0 +1,21 @@
+import test from 'tape';
+import { dom } from './helpers';
+import { Map } from 'immutable';
+import React from 'react';
+import ReactTestUtils from 'react/lib/ReactTestUtils';
+
+import House from '../components/house';
+
+const house = Map({ url: 'www.rightmove.co.uk/property-to-rent/property-46665035.html' });
```

Comment:
just bump the url key onto a new line? Given it will prob end up with more properties further down the round